src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -194,6 +196,29 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
   const [virtualRuneDepositAmountFiatUserCurrency, setVirtualRuneDepositAmountFiatUserCurrency] =
     useState<string | undefined>()
 
+  const [isUnsafeQuoteNoticeDismissed, setIsUnsafeQuoteNoticeDismissed] = useState<boolean | null>(
+    null,
+  )
+
+  const handleAcknowledgeUnsafeQuote = useCallback(() => {
```

Comment:
Seems like we should align on the unsafe ux flow from a product perspective. Do we do this explicit ack step for all unsafe actions? Is a warning enough without explicit ack? One reason I personally am not as big of a fan of this current setup is the unsafe slippage state supersedes the standard disabled states like not enough balance (maybe halted?), which you would need to ack to see disabled/error status

Reply:
Very valid point. Agreed with the concerns and that can also become tricky when you want to handle multiple unsafe states, which, IMO we shoud all handle with explicit ack. Most importantly, as you've noted, we're now also having very similar, yet slightly different unsafe states in different domains.

I think that's a @reallybeard and @twblack88 q but I believe this would be a good contender for a first dive into https://github.com/shapeshift/web/issues/6446 from both a product and eng. perspective.

Re: `is the unsafe slippage state supersedes the standard disabled states like not enough balance (maybe halted?), which you would need to ack to see disabled/error status` that's absolutely valid, and something we could use 6446 to tackle with a higher-order version of the button, which would only display the unsafe state *if* we're ready (meaning we're not loading, and there's no other error state).

Reply:
Happy for this to be addressed in a more formal manner through product and wait for specific actionable tickets as a follow up.

Reply:
https://github.com/shapeshift/web/issues/6605